java/org.travlyn.server/src/main/java/org/travlyn/server/externalapi/access/OpenRouteRequest.java

```diff
@@ -19,18 +22,46 @@
 
 public class OpenRouteRequest implements Request<Set<StopEntity>> {
     private static final String BASE_URL = "https://api.openrouteservice.org/pois";
+    private static final HashSet<Integer> excludedCategories = new HashSet<>();
 
-    private Gson gson = new Gson();
-    private HashMap<Integer, CategoryEntity> categoryList = new HashMap<>();
+    private final Gson gson = new Gson();
+    private Map<Integer, CategoryEntity> categoryList;
+
+    private final double latitude;
+    private final double longitude;
+    private final CityEntity city;
+
+    public OpenRouteRequest(double latitude, double longitude, CityEntity city, Map<Integer, CategoryEntity> categoryList) {
+        //set Categories that should be excluded (necessary due to filter limitations at ORS)
+        excludedCategories.add(136);
+        excludedCategories.add(237);
+        excludedCategories.add(238);
+        excludedCategories.add(231);
+        excludedCategories.add(234);
+        excludedCategories.add(261);
+        excludedCategories.add(262);
+        excludedCategories.add(267);
+        excludedCategories.add(271);
+        excludedCategories.add(280);
+        excludedCategories.add(282);
+        excludedCategories.add(283);
+        excludedCategories.add(288);
+        excludedCategories.add(290);
+        excludedCategories.add(293);
+        excludedCategories.add(297);
+        excludedCategories.add(298);
+        excludedCategories.add(302);
+        excludedCategories.add(303);
+        excludedCategories.add(307);
+        excludedCategories.add(623);
+        excludedCategories.add(624);
+        excludedCategories.add(626);
```

Comment:
This can be static: `private static final HashSet<Integer> excludedCategories = new HashSet<>(Arrays.asList(136, 237, 238, ...));`